util/libcrypto.num

```diff
@@ -4501,3 +4501,4 @@ EVP_sha512_256                          4442	1_1_1	EXIST::FUNCTION:
 EVP_sha512_224                          4443	1_1_1	EXIST::FUNCTION:
 OCSP_basic_sign_ctx                     4444	1_1_1	EXIST::FUNCTION:OCSP
 RAND_DRBG_bytes                         4445	1_1_1	EXIST::FUNCTION:
+X509_get0_authority_key_id              4446	1_1_1	EXIST::FUNCTION:
```

Comment:
Since this is going to be backported to 1.1.0, this is wrong. It think it should say 1_1_0h. Not sure about the numbers.

Reply:
dang, you are right.

Reply:
I have rebased this pull request on top of current master, and switched the version number of X509_get0_authority_key_id  to 1_1_0h.